manim/mobject/coordinate_systems.py

```diff
@@ -151,7 +178,23 @@ def get_parametric_curve(self, function, **kwargs):
         graph.underlying_function = function
         return graph
 
-    def input_to_graph_point(self, x, graph):
+    def input_to_graph_point(self, x: float, graph: ParametricFunction) -> np.ndarray:
+        """Returns the y-coordinate for a given curve and x-value.
```

Comment:
It's not just the y-coordinate but the coordinates of the point on the graph.
```suggestion
        """Returns the coordinates of point on the ``graph`` 
        corresponding to the input ``x`` value.
```